package.json

```diff
@@ -10,19 +10,33 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@mantine/core": "^6.0.13",
+    "@mantine/hooks": "^6.0.13",
+    "@tabler/icons-react": "^2.22.0",
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.1",
```

Comment:
did you install `match-sorter` it or it came with vite?

Reply:
I didn't install it